app/controllers/qa/linked_data_terms_controller.rb

```diff
@@ -186,7 +186,12 @@ def format
     end
 
     def jsonld?
-      format == 'jsonld'
+      format.casecmp('jsonld').zero?
+    end
+
+    def context?
+      context = params.fetch(:context, 'false')
+      context.casecmp('true').zero?
```

Comment:
Switched to use casecmp based on rubocop recommendation.  Applied the same change to jsonld? for consistency.  

And one other small cleanup to check for blank? instead of nil? for subauth, which covers more potential non-existing values for subauth.